src/Corax/Utils/EntryTermsReader.cs

```diff
@@ -425,6 +426,11 @@ public string Debug(Dictionary<long, string> fields = null)
 
         return sb.ToString();
     }
+
+    public void Dispose()
+    {
+        _llt.ReleaseCompactKey(ref Current);
```

Comment:
Is it okay to release it even it it was provided externally via constructor?

Reply:
I think we mostly pass it like below:

```
using var _ = _indexSearcher.Transaction.LowLevelTransaction.AcquireCompactKey(out var existingKey);

        for (var processingId = 0; processingId < matches.Length; ++processingId)
        {
            _indexSearcher.GetEntryTermsReader(matches[processingId], ref p, out var entryTermsReader, existingKey);
```
       
 so it seems it might be released twice

Reply:
Will check all instances of it.